linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 
-env.LISAV2_REMOTE = "https://github.com/lis/LISAv2.git"
-env.LISAV2_BRANCH = "master"
+env.LISAV2_REMOTE = "https://github.com/ader1990/LISAv2-1.git"
```

Comment:
we can make it as parameter, then can change into any repo we want?

Reply:
this is not needed, as this is just a work in progress to actually track a very hard to bisect bug.